codegen.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  schema: "https://graphql-pokemon2.vercel.app/",
+  documents: ["./**/*.graphql"],
```

Comment:
This glob matches all files in our codebase that are `.graphql` file. These are the queries/fragments/mutations that will have types/hooks auto-generated for us.